contributorQualification.md

```diff
@@ -0,0 +1,62 @@
+# Aims
+
+Our most important aim with the trusted setup is to avoid all practical possibility of a single actor controlling the contributions. A secondary aim is to avoid censoring anyone who wishes to contribute. 
+
+Our chosen approach is to encourage a large number of participants, with the expectation 
+that collusion, coercion, or interference is increasingly improbable with larger numbers of contributors. 
+Having a low barrier to entry with regard to the technical requirements and time-and-attention requirements means we draw from a large pool of potential candidates.
+
+A conflict arises between the above aims when we decide to defend against sybil operators using exclusionary criteria. 
+We will inevitably exclude some interested and genuine participants who don’t meet the criteria. The overriding aim is 
+to avoid attempts by malicious operators controlling large numbers of sybil identities. A balanced approach will allow 
+a large number of unique contributors and allow the possibility of a small number of sybil identities. 
+
+# Contribution Flow
+
+* Contributors will log in using one of the accepted authentication methods
+* The chosen identifier will be validated. Only valid identifiers will be able to progress to the following steps
+* A JWT will be issued in response to the login request. 
+* The JWT will be required in request headers for all subsequent requests. The token will be valid for a period of several hours. The contributor should not need to go through login again before
+completing their contribution, provided they do not log off or close the page.
+
+# Authentication Methods
+
+* Ethereum accounts 
+* GitHub 
+
+## Common Criteria
+
+Identifiers must not have previously submitted a contribution to this ceremony, nor attempted a submission. 
+
+## Authentication by ETH Account.
+### Identifiers: Eth accounts and ENS names
+
+Participants will be identified using Ethereum accounts. We have no need to prohibit pseudonymous contributions, given that they have already passed anti-sybil checks.
+However, well-known names may add to the credibility of the ceremony. For any address with an ENS name reverse-lookup configured, the ENS name will be used in place of the address. 
+
+Identifiers will be added to the transcript and will become part of the permanent record of the ceremony. 
```

Comment:
Ok, this is currently not a part of the specs. I will need to modify the `transcript.json` to allow this.